src/main/java/nl/hsac/fitnesse/util/iban/FRIbanGenerator.java

```diff
@@ -50,11 +50,13 @@ private String encodeAccount(String account) {
 
         char[] letters = account.toCharArray();
         String result = "";
-        for (char letter : letters)
-            for (int i = 0; i < chars.length(); i++)
+        for (char letter : letters) {
```

Comment:
The loop refactor was something intellij suggested and I blindly followed. Braces are indeed good for readability.